doc/main.html

```diff
@@ -200,13 +255,17 @@ <h4>introduction section</h4>
 
         <p>The <code>id</code> attribute shall be present and equal to <code>sec-introduction</code>.</p>
 
+        <p>If no Introduction is required, the section shall not be present.</p>
+
         <p>The heading will be generated automatically.</p>
       </section>
 
       <section id="sec-scope-section">
-        <h4>Scope</h4>
+        <h4>Scope section</h4>
+
+        <p>The <dfn>Scope</dfn> contains the scope of the document.</p>
 
-        <p>The <dfn>scope section</dfn> contains the scope of the document.</p>
+        <p>The section shall be present as: </p>
```

Comment:
I do not think we can say "shall" since it can contain more than one `<p>`

Reply:
Changed as other sections for consistency.